src/Artsy/Relay/createEnvironment.ts

```diff
@@ -30,13 +39,21 @@ interface RelayEnvironment extends Environment {
 
 export function createEnvironment(config: Config = {}) {
   const { cache = {}, checkStatus, user, relayNetwork } = config
-  const isServer = typeof window === "undefined"
+
   const relaySSRMiddleware = isServer
     ? new RelayServerSSR()
     : new RelayClientSSR(cache)
 
   relaySSRMiddleware.debug = false
 
+  // Only log on the client during development
+  let loggingEnabled = isDevelopment && !isServer
+
+  // Add client window hook to debug logging in prod
+  if (!isServer) {
+    window._toggleRelayLogging = () => (loggingEnabled = !loggingEnabled)
+  }
```

Comment:
@alloy figured its time to turn off logging in prod since things have settled a bit post /artist launch but wanted a way to toggle on if need be